docs/api.md

```diff
@@ -0,0 +1,3827 @@
+# ArduinoBLE library
+
+## BLE class
+
+Used to enable the BLE module.
+
+### `begin()`
+
+Initializes the BLE device.
+
+#### Syntax
+
+```
+BLE.begin()
+
+```
+
+#### Parameters
+
+None
+
+Returns
```

Comment:
"Returns" should be a H4 (four hashtags). Looks like this needs to be fixed in the rest of the file too!

Reply:
done